src/specs/components/timeline/demo/data.js

```diff
@@ -8,100 +8,73 @@ module.exports = {
     from: 5,
     to: -2,
   },
-  button: {
-    label: 'Show all timeline',
-    icon: {
-      path: '/icons.svg',
-      name: 'corner-arrow',
-      size: 'xs',
-      transform: 'rotate-180',
-    },
-    variant: 'secondary',
-    type: 'button',
-  },
   icon_path: '/icon.svg',
   items: [
     {
-      id: '0',
```

Comment:
a bit confused... i guess that these ids where requested, indeed we still support the id for the single items, but shouldn't it behave lke the others id we set? I mean that in this pr we are always setting a default for them, here we are not and we are not demoing or testing the id anymore, i think we should set a default as in the other cases, so it will always be there and the user can choose his own ids by passing it in the data.